src/DIRAC/FrameworkSystem/private/standardLogging/LoggingRoot.py

```diff
@@ -71,11 +67,12 @@ def __init__(self):
     def initialize(self, systemName, cfgPath, forceInit=False):
         """
         Configure the root Logging.
-        It can be possible to :
-        - attach it some backends : LogBackends = stdout,stderr,file,server
-        - attach backend options : BackendOptions { FileName = /tmp/file.log }
-        - add colors and the path of the call : LogColor = True, LogShowLine = True
-        - precise a level : LogLevel = DEBUG
+        It can be possible to:
+        - attach some backends to it: LogBackends = stdout,stderr,file,server
```

Comment:
```suggestion
        
        - attach some backends to it: LogBackends = stdout,stderr,file,server
```
empty line before bullet list
https://dirac--6708.org.readthedocs.build/en/6708/CodeDocumentation/FrameworkSystem/private/standardLogging/LoggingRoot.html#DIRAC.FrameworkSystem.private.standardLogging.LoggingRoot.LoggingRoot.initialize